docs/api/navigators/StackNavigator.md

```diff
@@ -128,6 +128,22 @@ All `navigationOptions` for the `StackNavigator`:
   - `tintColor` - Tint color for the header
 - `cardStack` - a config object for the card stack:
   - `gesturesEnabled` - Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android
+- `transitionConfig` - a function returning an object that will override default animations in `CardStack`.
+Object has optional keys `transitionSpec` and `screenInterpolator`. Example:
+```js
+transitionConfig: () => ({
```

Comment:
Fixed docs so that `transitionConfig` is actually part of `StackNavigatorConfig` and not `navigationOptions`.